tests/phpunit/tests/rest-api/rest-themes-controller.php

```diff
@@ -471,6 +475,22 @@ public function test_theme_requires_wp() {
 		$this->assertSame( '5.3', $result[0]['requires_wp'] );
 	}
 
+	public function test_theme_is_block_theme() {
+		switch_theme( 'block-theme' );
+
+		$response = self::perform_active_theme_request();
+		$result   = $response->get_data();
+
+		$this->assertTrue( $result[0]['is_block_theme'] );
```

Comment:
I would check if this key exists first.

Reply:
Thanks for the suggestion! Updated in https://github.com/WordPress/wordpress-develop/pull/4323/commits/ccd8533d4d9337f8b8e44ac773498d6e4bf25054